Source/WebKit/Scripts/webkit/tests/TestSerializedType.serialization.in

```diff
@@ -291,3 +292,8 @@ header: "RemoteVideoFrameIdentifier.h"
     IPC::ObjectIdentifierReference<WebKit::RemoteVideoFrameIdentifier> reference();
     uint64_t pendingReads();
 };
+
+header: "RemoteVideoFrameIdentifier.h"
+[Alias=class IPC::ObjectIdentifierReadReference<RemoteVideoFrameIdentifier>, AdditionalEncoder=StreamConnectionEncoder, CustomHeader] alias WebKit::RemoteVideoFrameReadReference {
```

Comment:
I don't see a need to add this particular test.  This file exists to test changes to the python, and this PR has no changes to the python that need testing.

Reply:
Same.

If you want to explicitly test serialization of a type, you do it in TestIPC's IPCSerialziation. But we're mostly only doing that for the much rarer obj-c types.

These identifiers are exercised by *all* of the testing infrastructure already, so I'm not worried about explicit tests for them.